Initialize google analytics on Academix

Piumal1999 opened the pull request:
## Purpose
<!--- Describe the problems, issues, or needs driving this feature/fix and include links to related issues -->
The purpose of this PR is to fix #55 

## Goals
<!---  Describe the solutions that this feature/fix will introduce to resolve the problems described above -->
Implement a way to track user's activities

## Approach
<!--- Describe how you are implementing the solutions. Include a link to a Markdown file or Google doc if the feature write-up is too long to paste here. -->
Initialize Google analytics
Created a util for track views


![Screencast from 2020-07-27 13_46_46](https://user-images.githubusercontent.com/27498587/88520354-185d2380-d011-11ea-9b3f-2d1f4b52f432.gif)


##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
<!--- List any other related PRs --> 
N/A

## Test environment
<!--- List all JDK versions, operating systems, databases, and browser/versions on which this feature/fix was tested --> 

## Learning
<!--- Describe the research phase and any blog posts, patterns, libraries, or add-ons you used to solve the problem. -->
https://medium.com/google-cloud/tracking-site-visits-on-react-app-hosted-in-google-cloud-using-google-analytics-f49c2411d398

jayasanka-sack commented on app/src/components/App/index.tsx:
```diff
@@ -15,6 +16,9 @@ import {
   MenuOutlined,
 } from '@ant-design/icons';
 
+const trackingId = 'UA-167873271-2';
+ReactGA.initialize(trackingId);
```
```suggestion
ReactGA.initialize('UA-167873271-2');
```

jayasanka-sack replied:
Also, add a comment before this line.

Piumal1999 replied:
Done

jayasanka-sack commented on app/src/services/util/track.js:
```diff
@@ -0,0 +1,5 @@
+import ReactGA from 'react-ga';
+
+export const trackPage = () => {
```
Shall we refactor the function name?

something like `trackPageWithGoogleAnalytics`

jayasanka-sack replied:
Lets change the file name to `googleAnalytics.js`

Piumal1999 replied:
Okay. Updated

Piumal1999 commented:
> WoW! Great job @Piumal1999 !!
> 
> I've added some comments. Apart from that, is this going to update when we run this on locally? If yes, is there a way to prevent it?

Yeah. I added a filter from the google analytics dashboard. Now it'll only update using data taken from sefglobal.org hostname.
